app/pages/lab/project.cjsx

```diff
@@ -53,13 +54,25 @@ EditProjectPage = createReactClass
     linkParams =
       projectID: @props.project.id
 
+    thisProjectUsesFEM = isThisProjectUsingFEMLab @props.project, @props.location
+
+    projectLink = "/projects/#{@props.project.slug}"
+    if thisProjectUsesFEM then projectLink = "#{FEM_LAB_PREVIEW_HOST}#{projectLink}"
```

Comment:
This is going to need to add `?env=staging` on staging.
eg. here the project preview link gives a 404 error.
https://pr-6053.pfe-preview.zooniverse.org/lab/908?femLab=true

Reply:
I guess this is complicated by the fact that `*.pfe-preview.zooniverse.org` uses the staging API, but `frontend.preview.zooniverse.org` uses the production API.